app/models/dialog.rb

```diff
@@ -61,6 +61,11 @@ def validate_children
       errors.add(:base, _("Dialog %{dialog_label} must have at least one Tab") % {:dialog_label => label})
     end
 
+    duplicated_field_names = duplicate_dialog_fields_names(dialog_fields)
+    unless duplicated_field_names.empty?
```

Comment:
Prefer the positive case `if duplicated_field_names.present?`  (I'm assuming this is always an Array, but may be blank)